spec/forms/flood_risk_engine/steps/local_authority_address_form_spec.rb

```diff
@@ -58,35 +40,33 @@ def form_requires_address_lookup
         }
 
         it "is valid when valid UK UPRN supplied via drop down but no street_address present" do
-          form_requires_address_no_street_lookup do
-            expect(form.validate(valid_but_no_street_address)).to eq true
-          end
+          mock_ea_address_lookup_find_by_uprn
+          expect(form.validate(valid_but_no_street_address)).to eq true
```

Comment:
I think we need a smarter mock for this as the required params are 

let(:valid_but_no_street_address) {
          {
            "#{form.params_key}": { post_code: "HX3 0TD", uprn: "10010175140" }
          }
        }